.travis/molecule.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Avoid lint error
+sed -i "s/{%/# {%/g" roles/postgres/templates/postgres.deployment.yaml
+
+# Avoid nodePort error:
+# nodePort: Invalid value: 24816 The range of valid ports is 30000-32767
+# nodePort: Invalid value: 24817 The range of valid ports is 30000-32767
+find ./roles/* -exec sed -i 's/24816/31816/g' {} \;
+find ./roles/* -exec sed -i 's/24817/31817/g' {} \;
```

Comment:
I think we should look into how to configure molecule's k8s to allow a lower range. We do that for k3s in .travis/k3s-install.sh.

This fits into the broader theme of "molecule tests should pass before we push to quay.io, as part of the "test-all-components" tests.

Reply:
the k8s docker images comes from here:
https://github.com/bsycorp/kind
I couldn't find anything there referring to it

Reply:
This PR is already huge, wdyt about filing an issue and addressing it in other PR after merging this one?

Reply:
Let's just do that.

Reply:
filed: https://pulp.plan.io/issues/7107